heareval/tasks/pipeline.py

```diff
@@ -171,7 +171,9 @@ def slugify_file_name(relative_path: str):
 
         The basic version here takes the filename and slugifies it.
         """
-        return f"{slugify(str(Path(relative_path).stem))}"
+        slug_text = str(Path(relative_path).stem)
+        slug_text = slug_text.replace("-", "_negative_")
```

Comment:
Add a comment to explain why

Reply:
This should not be here. dcase should overwrite slugify_file_name like speech_commands does.

Reply:
i.e. this case of replacing "-" with "_negative_" is specific only to dcase